epicfocus/README.md

```diff
@@ -0,0 +1,129 @@
+
+Here is the visualization of our VSM and how it changed over time as we tweaked the process.
+![VSM](images/vsm.png?raw=true "Value Stream Map")
+
+By performing this exercise, you will identify the waste in your process, and it will become clear what parts of it need change. Of course, you still don't know what work items are valuable by themselves. For that, you need another metric.
+
+## Drilling down into the backlog with the Epic Focus Metrics
+#### 1. Identify value-adding and value-retaining backlog items
+By building a value stream map and eliminating waste, you can speed up your processes. We would like to offer you a similar exercise only performed not against activities, but against your backlog items. This exercise can give you insight into the speed of your feature delivery and your focus on value.
+
+Put all the completed work into two buckets: value-adding and value-retaining. We chose to mark new features, or epics, as value-adding and bugs and technical debt items as value-retaining. Both are important; we need to add new value while maintaining the existing value of our products.
+
+![Distribution of tickets](images/adding-vs-retaining.png?raw=true "Distribution of work items")
+
+Once you have a distribution of work, e.g. the ratio between value-adding and value-retaining items, you can drill down on each of them and get even more insights. In our situation, we see that adding new value seems to have less priority than maintaining the existing value. It could mean that there is a lot of bugs, and there is a quality problem, or perhaps, it is a [tech debt](https://xebia.com/blog/using-metrics-to-find-the-pain-points-in-a-legacy-codebase/) that's holding them back and doesn't allow extending the product. Having a lot of value-retaining work might create a perception of slowness in the eyes of the stakeholders.
+
+#### 2. Calculate the feature delivery speed
+As a next step, we decided to look deeper into the value-adding bucket. We used features as value-adding work. For each feature, we identified the duration (in days) and the number of work items completed within that period. By dividing the number of completed tickets by duration, we calculated the time to market of the feature.
+
+![Feature delivery speed](images/feature-speed.png?raw=true "Feature time to market")
```

Comment:
I don't understand what the "Total speed" is on the picture. Is it the sum of the speed for each individual feature? If so, then it does not add up.

Reply:
Yes, It's not the sum.
When we moved it to WordPress we changed the line below the image to read:

`For comparison, we calculated the total delivery speed - the speed of the features development and other stuff combined.`

I think this should be enough clarification.

Reply:
To be even clearer, I also added a subtitle to the image explaining the total :))